pyccel/codegen/printing/pycode.py

```diff
@@ -273,6 +273,12 @@ def _print_PythonRange(self, expr):
                 stop  = self._print(expr.stop ),
                 step  = self._print(expr.step ))
 
+    def _print_PythonEnumerate(self, expr):
+        name = self._aliases.get(type(expr), expr.name)
+        return '{name}({elem})'.format(
+                name = name,
+                elem = self._print(expr.element))
```

Comment:
Could the built-in Python function `enumerate` get aliased?

Reply:
You are right, this was copied from range which also cannot be aliased. Fixed in 2ff1eda